src/stores/AccountStore/AccountStore.ts

```diff
@@ -63,6 +63,16 @@ export class AccountStore {
     return URL.createObjectURL(accountBlob);
   }
 
+  exportPrivateKey() {
+    if (!this.privateKey) {
+      throw new Error('No private key found!');
+    }
+    const privateKeyBlob = new Blob([JSON.stringify(this.privateKey)], {
```

Comment:
`this.privateKey` - is etherium private key, not the one used for Cere blockchain

Reply:
This is how the `ed` private key is created:
https://github.com/cere-io/cere-wallet-client/blob/dev/packages/wallet-engine/src/engine/polkadot.ts#L21